src/vivarium_csu_zenon/data/loader.py

```diff
@@ -65,29 +65,33 @@ def get_data(lookup_key: str, location: str) -> pd.DataFrame:
         project_globals.ISCHEMIC_STROKE.CSMR: load_standard_data,
         project_globals.ISCHEMIC_STROKE.RESTRICTIONS: load_metadata,
 
+        project_globals.DIABETES_MELLITUS.PREVALENCE: load_standard_data,
         project_globals.DIABETES_MELLITUS.MODERATE_DIABETES_PREVALENCE: load_diabetes_mellitus_prevalence,
         project_globals.DIABETES_MELLITUS.SEVERE_DIABETES_PREVALENCE: load_diabetes_mellitus_prevalence,
+        project_globals.DIABETES_MELLITUS.INCIDENCE_RATE: load_standard_data,
+        project_globals.DIABETES_MELLITUS.REMISSION_RATE: load_standard_data,
+        project_globals.DIABETES_MELLITUS.MODERATE_DIABETES_PROPORTION: load_diabetes_mellitus_proportion,
+        project_globals.DIABETES_MELLITUS.SEVERE_DIABETES_PROPORTION: load_diabetes_mellitus_proportion,
         project_globals.DIABETES_MELLITUS.MODERATE_DIABETES_DISABILITY_WEIGHT: load_diabetes_mellitus_disability_weight,
         project_globals.DIABETES_MELLITUS.SEVERE_DIABETES_DISABILITY_WEIGHT: load_diabetes_mellitus_disability_weight,
-        project_globals.DIABETES_MELLITUS.MODERATE_DIABETES_EMR: load_diabetes_mellitus_excess_mortality_rate,
-        project_globals.DIABETES_MELLITUS.SEVERE_DIABETES_EMR: load_diabetes_mellitus_excess_mortality_rate,
         project_globals.DIABETES_MELLITUS.CSMR: load_standard_data,
+        project_globals.DIABETES_MELLITUS.EMR: load_diabetes_mellitus_excess_mortality_rate,
```

Comment:
I think you should probably be able to use load_standard_data here unless you ran into an issue I'm not thinking of.

Reply:
It looks like that is correct, but i just did a test and what I'm doing produces identical results. Since I've already built the artifacts, I think I will leave it as is if that is ok.